package.json

```diff
@@ -78,6 +78,7 @@
     "randomstring": "^1.1.0",
     "rc": "~0.5.0",
     "request": "^2.51.0",
+    "request-promise": "^1.0.2",
```

Comment:
interesting -- i've just been using bluebird's `promisify` on request.get and request.post, so i'm not sure if we need to add this dependency